backend.py

```diff
@@ -440,6 +441,22 @@ def performInstallation(answers, ui_package, interactive):
                                "latest dedicated ISO." %
                                (answers['linstor-version'], ', '.join(available_linstor_versions)))
 
+    # if upgrading a host with xapi, check we can upgrade it first
+    if answers['install-type'] == INSTALL_TYPE_REINSTALL and answers['xapi-version']:
+        available_xapi_versions = repository.listPackagesFromRepos(
+            main_repositories, 'xapi-core', '%{evr}')
+        if not available_xapi_versions:
+            raise RuntimeError("No xapi package found")
```

Comment:
could be good to be more specific, like "... found in package source" (to avoid confusion with installed package)

Reply:
To me, the message should be either about `XAPI`, as the component name, or `xapi-core`, as the package name that we actually looked for.